src/components/Navbar/index.tsx

```diff
@@ -34,81 +35,45 @@ const Navbar = ({ isAuth }: any) => {
     }
   }
 
+  const isSmallScreen = useMediaQuery('(max-width:819px)')
+
   return (
     <>
-      <Box
-        sx={{
-          flexGrow: 1,
-          width: '100%',
-          overflow: 'hidden'
-        }}
-      >
-        <AppBar
-          position='fixed'
-          sx={{
-            backgroundColor: '#eeeeee',
-            color: '#280606',
-            top: 0,
-            width: '100%'
-          }}
-        >
-          <Toolbar>
+      <Box>
+        <AppBar position='fixed' sx={{ backgroundColor: '#eeeeee', color: '#280606' }}>
+          <Toolbar sx={{ display: 'flex', justifyContent: 'space-between' }}>
             <Typography
               variant='h5'
               component='div'
               sx={{
-                flexGrow: 1,
                 fontWeight: 'bold'
               }}
             >
               Trust Claims
             </Typography>
-            <Box
-              sx={{
-                display: 'flex',
-                justifyContent: 'center',
-                columnGap: 3
-              }}
-            >
+            {isSmallScreen ? (
+              isAuth ? (
+                <ProfileDropdown />
+              ) : (
+                <Responsive />
+              )
+            ) : (
               <>
-                <Paper
-                  component='div'
-                  sx={{
-                    p: '2px 4px',
-                    display: 'flex',
-                    alignItems: 'center',
-                    width: 400
-                  }}
-                >
-                  <InputBase
-                    type='search'
-                    value={searchVal}
-                    placeholder='Search a Claim'
-                    onChange={e => setSearchVal(e.target.value)}
-                    onKeyUp={handleSearchKeypress}
-                    sx={{
-                      ml: 1,
-                      flex: 1
-                    }}
-                  />
-                  <IconButton type='button' sx={{ p: '10px' }} aria-label='search' onClick={handleSearch}>
-                    <SearchIcon />
-                  </IconButton>
-                </Paper>
+                <SearchBar />
 
                 {isAuth && <ProfileDropdown />}
                 {!isAuth && (
-                  <>
-                    <Button color='inherit' onClick={() => navigate('/login')}>
+                  <Box>
+                    <Button sx={{ pr: '30px' }} color='inherit' onClick={() => navigate('/login')}>
                       Login
                     </Button>
-                    <Button color='inherit' onClick={() => navigate('/register')}>
+                    <Button sx={{ pr: '30px' }} color='inherit' onClick={() => navigate('/register')}>
                       Register
                     </Button>
-                  </>
+                  </Box>
                 )}
               </>
-            </Box>
+            )}
```

Comment:
This logic should be based around `isAuth`, instead of `isSmallScreen`. Think of it this way:
```
{isAuth && (
  <>
    {isSmallScreen && <Your components for small screen here>}
    {!isSmallScreen && <Your components for big screens here>}
    <Components that will be rendered regardless of screen size />
  </>
)}
{!isAuth && ...do similar things as above}
```